subprojects/dependency-management/src/test/groovy/org/gradle/internal/resource/transfer/ProgressLoggingExternalResourceAccessorTest.groovy

```diff
@@ -99,9 +99,9 @@ class ProgressLoggingExternalResourceAccessorTest extends Specification {
         then:
         1 * progressLoggerFactory.newOperation(_) >> progressLogger
         1 * progressLogger.started()
-        1 * progressLogger.progress('1 KB/4 KB downloaded')
-        1 * progressLogger.progress('3 KB/4 KB downloaded')
-        1 * progressLogger.progress('4 KB/4 KB downloaded')
+        1 * progressLogger.progress('1 KB/4 KB downloaded: location')
```

Comment:
Both. There's a ticket handled by @jjohannes about an intermediate build operation for "getting the files of" compile classpath (aka, a build operation when we actually get the files, not resolve the configuration, which is already done). And, we need the console to use build operations. Today, we have a hierarchy of build operations already, it's just not displayed.